### DISCUSSION TITLE
Clean Uninstall

### DISCUSSION THREAD

**Discussion by D0minatorX** [seerge/g-helper#2989] [Q&A]
How can I Uninstall G-helper completely? because When I delete and re-download the exe file, it will show the settings before the deletion.
I appreciate any help you can provide.

**Comment by seerge** [comment]
> [How do I uninstall G-helper?](https://github.com/seerge/g-helper/wiki/FAQ#how-do-i-uninstall-g-helper)
> G-helper is a single exe, and it doesn't install anything in the system. To remove it - you can simply delete exe :) If you have applied any custom fan profiles or PPTs - before removing I would recommend selecting your favorite performance mode (for example balanced) and clicking "Factory defaults" under Fans + Power.

**Reply by D0minatorX** [reply]
I tried this but still took the old settings, the only way I was able to go back to the default was by reinstalling Windows.

**Reply by seerge** [reply]
Then your question is not about uninstalling. 

Settings are stored in `%appdata%\GHelper\config.json` folder, you need to delete it if you want to reset settings.

https://github.com/seerge/g-helper/wiki/FAQ#where-can-i-find-app-settings-or-logs-

**Reply by D0minatorX** [reply]
Thanks, I will try and tell you; this should completely remove it and go back to the default settings from Asus right?

**Reply by seerge** [reply]
Yes, that will reset all changes you have done inside G-Helper

**Reply by Budeh75** [reply]
I deleted the folder %appdata%GHelper but when I ran GHelper.exe again, the changes I had made still persisted. I created a 'Game' performance mode and it still shows up even after deleting the settings folder.

**Reply by seerge** [reply]
@Budeh75 delete backup config at `C:\ProgramData\GHelper`
